setup.py

```diff
@@ -31,6 +31,16 @@
                 'caproto.threading',
                 'caproto.trio',
                 ],
+      entry_points={
+          'console_scripts': [
+              'caproto-get = caproto.sync.client:get_cli',
+              'caproto-put = caproto.sync.client:put_cli',
+              'caproto-monitor = caproto.sync.client:monitor_cli',
+              'caproto-repeater = caproto.sync.client:repeater_cli',
+              'caproto-defaultdictserver = caproto.examples.defaultdictserver:main',
+              'caproto-spoof-beamline = caproto.examples.spoof_beamline:main',
+          ],
+      },
       scripts=glob.glob('scripts/*'),
```

Comment:
If I am interpreting [the documentation](https://setuptools.readthedocs.io/en/latest/setuptools.html#automatic-script-creation) correctly, these `entry_points` work on POSIX too, so we can delete `scripts=`.